lib/apiservers/engine/proxy/mocks/image_mock.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.// Code generated by MockGen. DO NOT EDIT.
+// Source: image.go
+
+// Package mocks is a generated GoMock package.
+package mocks
+
+import (
+	context "context"
+	reflect "reflect"
+
+	gomock "github.com/golang/mock/gomock"
```

Comment:
I am extremely happy to see gomock in VIC project, but I do not see it in vendor on my side. Did you add it locally?

Reply:
btw, since this is a generated code, which means you should provide a way to regenerate it.